src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -850,10 +837,9 @@ public void deleteCourseCascade(String courseId) {
     }
     
     /**
-     * Returns a CSV for the details(name, email, status) of all students mapped to a given course.
      * @param courseId
      * @param instructorGoogleId
-     * @return a CSV String with student details
+     * @return a CSV for the details(name, email, status) of all students belongs to a given course
```

Comment:
return a CSV string of the details(name, email, status) of all students belonging to a given course